test/com/google/enterprise/adaptor/opentext/OpentextAdaptorTest.java

```diff
@@ -3049,6 +3067,520 @@ public void testDoEmail() throws IOException {
         responseMock.outputStream.toString("UTF-8"));
   }
 
+  @Test
+  public void testLastModifiedQuery() {
+    SoapFactoryMock soapFactory = new SoapFactoryMock();
+    OpentextAdaptor adaptor = new OpentextAdaptor(soapFactory);
+    AdaptorContext context = ProxyAdaptorContext.getInstance();
+    Config config = initConfig(adaptor, context);
+    config.overrideKey("opentext.src", "1234,5678");
+    adaptor.init(context);
+
+    String query = adaptor.getLastModifiedQuery();
+    assertTrue(query.indexOf("Location_ID1=1234") != -1);
+    assertTrue(query.indexOf("Location_ID2=5678") != -1);
+    assertTrue(
+        query.indexOf(
+            "QLREGION+%22OTModifyTime%22%5D+%3E+%22000000%22") != -1);
+  }
+
+  @Test
+  public void testGetXmlSearchCountResults() throws Exception {
+    String response =
+        "<?xml version=\"1.0\" encoding=\"UTF-8\"?>"
+        + "<Output>"
+        + "  <SearchResultsInformation>"
+        + "    <CurrentStartAt>1</CurrentStartAt>"
+        + "    <NumberResultsThisPage>23</NumberResultsThisPage>"
+        + "  </SearchResultsInformation>"
+        + "</Output>";
+    xmlSearchCount(response, 23);
```

Comment:
Maybe change this to assertXmlSearchCount(23, response), so I don't have to go find the helper method to understand what's going on, and to put the expected value first in an assertion as usual?